src/content/docs/en/pages/main-menu/reference/marketplace/marketplace-seller-guide.mdx

```diff
@@ -1,141 +1,217 @@
+- A software package that is instantiated and works with the underlying technologies of Azion's platform.
+- An edge function that can be triggered by specific events and execute specific tasks.
+- A template with a preset of rules for any Azion product.
 
-The following is general pricing information about products in Azion Marketplace. All pricing is based on US dollars (USD).
+:::caution[Warning]
+All Terms and Conditions are included in the [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/). You must agree with this document to offer your products on Azion Marketplace.
+:::
 
-* There is no service fee for Bring Your Own License (BYOL) or  Software free of charge products on Azion Marketplace.
-* To deliver on our customer promise of selection, we require that all BYOL products also have a paid option. This is so that customers who don’t have existing licenses have the option to purchase and use the products.
-	* For BYOL products, we realize that the online purchase of software is a departure from how some companies do business. In light of this, for the first 90 days after launch we will relax the requirement that this software is accompanied by a version available for purchase on the Azion Marketplace. During this time, the Azion Marketplace account management teams will work with you to address challenges and to determine if and how the software can be made available for purchase on the Azion Marketplace.
-* For Paid products, the seller defines the charges for using the software and Azion Marketplace collects software charges from the customer.
-* All seller's products will incur associated Azion charges depending on the services used. These rates and fees are defined and controlled by Azion. They are displayed separately on the Azion Marketplace detail pages to help customers understand the potential cost of using the products.
+---
+
+## Marketplace business model
+
+**Azion Marketplace** operates with the following business models:
+
+- **Free model**: buyers can install the product on the go, and pay only when they use it. Subject to terms of use defined.
+- **Bring your own License (BYOL)**: buyers must bring their current license or credentials from the seller's platform, as well as provide valid credentials to authenticate their account and connect Azion with the seller's platform.
+
+:::note
+Azion will charge buyers for the Azion services costs generated by the usage, whether in the free or BYOL model. In case of charges defined by the seller in a BYOL product, it becomes part of the buyer’s bill and, after payment, Azion Marketplace pays the seller.
+:::
+
+Azion requires that all BYOL products also have a [paid option](#pricing). This is so that users who don’t have existing licenses have the option to purchase and use the products.
```

Comment:
```suggestion
Azion requires that all BYOL products also have a [paid option](#pricing) in case users who don’t have existing licenses want to purchase and use the products.
```